spec/features/hotel/show_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+describe 'extension' do
+  # Extension
+
+  # As a visitor,
+  # When I visit a hotel's show page,
+  # Then I see a unique list of all guests that have stayed at this hotel.
+  it 'has a list of all unique guests' do
+    budapest = Hotel.create!(name: "Grand Budapest", location: 'Turkey')
+    room1 = Room.create!(hotel_id: budapest.id, suite: "Presidential", rate: 125)
+    charlize = Guest.create!(name:"Charlize Theron", nights: 3)
+    danny = Guest.create!(name:"Danny", nights: 2)
+    mark = Guest.create!(name:"Mark", nights: 3) 
+    guest_room1 = GuestRoom.create!(room_id: room1.id, guest_id: charlize.id)
+    guest_room2 = GuestRoom.create!(room_id: room1.id, guest_id: danny.id)
+    guest_room3 = GuestRoom.create!(room_id: room1.id, guest_id: mark.id)
+    guest_room4 = GuestRoom.create!(room_id: room1.id, guest_id: charlize.id)
+
+    visit "/hotels/#{budapest.id}"
+    expect(page).to have_content("#{budapest.name} Show Page")
+    expect(page).to have_content("List of Guests")
+    expect(page).to have_content(charlize.name, count: 1)
+    expect(page).to have_content(danny.name)
+    expect(page).to have_content(mark.name)
+  end
+end
```

Comment:
I know that during an IC, time is of the essence but I would like to see some better test set up. `describe 'extension'` is not descriptive of what this feature test is for, this is a test for the hotel show page, and should be in the `describe`. I would also like to see some additional `describe` blocks being used to set up your tests.

`it 'has a list of all unique guests'` is a better description of what is being expected in the test compared to your guests show spec.

Beyond that, the tests are set up perfectly, and I like how you used string interpolation in your assertions to give them a little more weight and less ambiguity.

The only recommendation other recommendation I would make is to start working in `within` blocks into your tests.